packages/wrangler/src/api/startDevWorker/DevEnv.ts

```diff
@@ -146,10 +148,16 @@ function createWorkerObject(devEnv: DevEnv): Worker {
 			return devEnv.proxy.ready.promise.then(() => undefined);
 		},
 		get url() {
-			return devEnv.proxy.ready.promise.then((ev) => ev.url);
+			return devEnv.proxy.ready.promise.then((ev) => {
+				assert(ev);
```

Comment:
It is good practice to add a description parameter to `assert()` calls so that we get a helpful message if the assertion fails.

Reply:
```suggestion
				assert(ev, "Expected the ready promise to resolve to a defined value but got " + ev);
```